src/App.tsx

```diff
@@ -15,36 +16,99 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  SORT_BY_ALPHABET = 'alphabet',
+  SORT_BY_LENGTH = 'length',
+}
+
+interface Goods {
```

Comment:
```suggestion
interface Options {
```
This interface name is misleading
Or Config/OrderOptions, etc